test/Java10andUp/src/org/openj9/test/stackWalker/StackWalkerTestJava10.java

```diff
@@ -22,45 +22,93 @@
 package org.openj9.test.stackWalker;
 
 import java.util.HashMap;
+import java.util.Iterator;
 import java.util.List;
+import java.util.Optional;
 import java.util.stream.Collectors;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+import static org.testng.Assert.fail;
+
 import java.lang.StackWalker.Option;
 import java.lang.StackWalker.StackFrame;
+import java.lang.invoke.MethodType;
 
 import org.testng.annotations.Test;
 import org.testng.log4testng.Logger;
 
-@Test(groups = { "level.extended" })
+/**
+ * This test StackWalker features added in Java 10 and is supplementary to the Java 9 tests.
+ *
+ */
 @SuppressWarnings("nls")
```

Comment:
Don't use `@SuppressWarnings`.  Can you add the NON-NLS message to the descriptors instead?